File: CursoP00_Uber/JS/Car.js
// en la misma clase genero el constructor
function Car(license, driver){
    this.license = license;
    this.driver = driver;
    this.id;
    this.passenger;
}

//Metodo para imprimir
Car.prototype.printDataCAr = function(){
    console.log(this.license)
    console.log(this.driver.name)
    console.log(this.driver.document)
}

File: CursoP00_Uber/Java/Car.java
class Car {
    Integer id;
    String license;
    Account driver;
    Integer passenger;

    //Metodo Constructor
    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver; 
    }

    void printDataCar() {
        System.out.println("Car plates: " + license + ", name: " + driver.name + ", document: "+ driver.document);
    }
}
